specification/definitions/Shipment.json

```diff
@@ -152,6 +152,82 @@
             }
           }
         },
+        "customs": {
+          "type": "object",
+          "properties": {
+            "package_type": {
+              "type": "string",
+              "enum": [
+                "merchandise",
+                "sample_merchandise",
+                "returned_merchandise",
+                "documents",
+                "gifts"
+              ],
+              "example": "merchandise"
+            },
+            "invoice": {
+              "type": "string",
+              "example": "9000"
+            },
+            "items": {
+              "type": "array",
+              "items": {
+                "type": "object",
+                "required": [
+                  "sku",
+                  "description",
+                  "item_value",
+                  "quantity",
+                  "hs_code",
+                  "origin_country_code"
+                ],
+                "additionalProperties": false,
+                "properties": {
+                  "sku": {
+                    "type": "string",
+                    "example": "123456789"
+                  },
+                  "description": {
+                    "type": "string",
+                    "example": "Kittens"
```

Comment:
:open_mouth: 

Misschien hier iets realistischers in zetten dat je op de post doet. :stuck_out_tongue_closed_eyes: 

`xbox one x`?